library/network/src/modules/NetworkInterfaces.rb

```diff
@@ -681,6 +683,20 @@ def ConcealSecrets(devs)
       deep_copy(out)
     end
 
+    # Get current sysconfig configured interfaces
+    #
+    # @param devregex [String] regex to filter by
+    # @return [Array] of ifcfg names
+    def get_devices(devregex = "[~]")
```

Comment:
why parameter is not regexp and instead only string that is later interpreted as regexp?